hub/core/lock.py

```diff
@@ -44,22 +44,35 @@ def __init__(self, storage: StorageProvider, path: str):
             self.username = username
 
     def acquire(self, timeout=10, force=False):
+        start_time = time.time()
```

Comment:
could you please add here some more comments, so that one could understand what is the main purpose of this function.